### PR TITLE
Add function to return form value types

### PR DESCRIPTION
**Pull request by bricesanchez**
@parndt could you release the 4.0.0 version after this merge and release the 3.0.1 version based on the https://github.com/refinery/refinerycms-settings/tree/3-0-stable branch ?

Thanks @veger and @parndt !

### CODE REVIEW

File: app/models/refinery/setting.rb [hunk 1]
```diff
@@ -30,6 +30,10 @@ def per_page
         12
       end
 
+      def form_value_types
```
Comment (parndt) [inline app/models/refinery/setting.rb]: you might want to update the form to use this version so that if it's overridden then it does something.

Comment (bricesanchez) [reply app/models/refinery/setting.rb]: Hahaha! You're right i've forgot to use it in the view.

Comment (bricesanchez) [reply app/models/refinery/setting.rb]: That's fixed now.

**Comment by bricesanchez** [comment]
@parndt Could you release the gems?

**Comment by bricesanchez** [comment]
@parndt Did you forget to release 3.0.1 ? :)

**Comment by parndt** [comment]
@bricesanchez nope, I had a couple of local issues and have rescheduled that job.

**Comment by bricesanchez** [comment]
ok

**Pull request merged**